src/magento/change-log.md

```diff
@@ -4,6 +4,15 @@ title: Change Log
 
 Our documentation is continually updated with new topics, clarifications, and corrections to existing content. Learn more about new features, major updates, and releases, organized by month and year. Check back every now and then to see what's new.
 
+## May 2020
```

Comment:
this will conflict with the change log update published this morning.  may want to refresh and add your table changes after.

Reply:
ok, thanks